projects/1.19/assets/yellowbrosss-extras/yellowbrossextras/lang/zh_cn.json

```
@@ -0,0 +1,41 @@
+{
+    "item.yellowbrossextras.defender_spawn_egg": "\u00A7d防御者刷怪蛋（WIP）",
+    "item.yellowbrossextras.sneaker_spawn_egg": "潜行苦力怕刷怪蛋",
+    "item.yellowbrossextras.paracreeper_spawn_egg": "空降苦力怕刷怪蛋",
+    "item.yellowbrossextras.crawler_spawn_egg": "爬行苦力怕刷怪蛋",
+    "item.yellowbrossextras.freaker_spawn_egg": "怪异苦力怕刷怪蛋",
+    "item.yellowbrossextras.sprayer_spawn_egg": "扫射苦力怕刷怪蛋",
+
+    "entity.yellowbrossextras.defender": "防御者",
+    "entity.yellowbrossextras.sneaker": "潜行苦力怕",
+    "entity.yellowbrossextras.paracreeper": "空降苦力怕",
+    "entity.yellowbrossextras.crawler": "爬行苦力怕",
+    "entity.yellowbrossextras.freaker": "怪异苦力怕",
+    "entity.yellowbrossextras.sprayer": "扫射苦力怕",
+
+    "subtitles.entity.defender.hurt": "防御者：受伤",
+    "subtitles.entity.defender.step": "防御者：移动",
+    "subtitles.entity.defender.jump": "防御者：跳跃",
+    "subtitles.entity.defender.saw": "电锯：切割",
+    "subtitles.entity.defender.saw_start": "防御者：使用电锯",
+    "subtitles.entity.defender.sword_whoosh": "防御者：挥剑",
+    "subtitles.entity.defender.sword_hit": "防御者：挥剑命中某物",
+    "subtitles.entity.defender.boomerang_hit": "回旋镖：命中",
+    "subtitles.entity.defender.spin_short": "防御者：旋转",
+    "subtitles.entity.defender.crash": "防御者：碰撞",
+    "subtitles.entity.defender.spike": "尖刺",
+    "subtitles.entity.defender.shoot": "防御者：投射",
+    "subtitles.entity.defender.chainsaw_warn": "电锯：启动",
+    "subtitles.entity.defender.chainsaw": "电锯：轰鸣",
+    "subtitles.entity.defender.smack": "防御者：拳打脚踢",
+    "subtitles.entity.defender.phase_ended": "防御者：阶段转换",

```

```suggestion
    "subtitles.entity.defender.phase_ended": "防御者：受到重创",
```
按照英文文件的value来翻可能合理一些